update-affiliated/update_astropy_helpers.py

```diff
@@ -42,6 +49,14 @@
 *This is intended to be helpful, but if you would prefer to manage these
 updates yourself, or if you notice any issues with this automated update,
 please let {1} know!*
+
+
+Along with with the astropy core release v3.0, we started to release the
```

Comment:
we started to release the astropy-helpers v3.0.x versions, too => we have also started to release astropy-helpers v3.0.x versions